script.js

```diff
@@ -1,4 +1,118 @@
+var win_count = 0;
+var loss_count = 0;
+var name_input = "";
+
+var random_pss = function () {
+  var paper = "Paper";
+  var scissors = "Scissors";
+  var stone = "Stone";
+  //random numbers from 1 - 3
+  var choices = Math.floor(Math.random() * 3) + 1;
+  // randomize the paper scissors stone
+  if (choices == 1) {
+    return paper;
+  } else if (choices == 2) {
+    return scissors;
+  } else if (choices == 3) {
+    return stone;
+  }
+};
+
+var process = function (mypov, computer_pov) {
+  //normal stream
+
+  if (mypov == "Scissors" && computer_pov == "Paper") {
+    win_count += 1;
+    return "you win!";
+  } else if (mypov == "Scissors" && computer_pov == "Stone") {
+    loss_count += 1;
+    return "you lose!";
+  } else if (mypov == "Paper" && computer_pov == "Stone") {
+    win_count += 1;
+    return "you win!";
+  } else if (mypov == "Paper" && computer_pov == "Scissors") {
+    loss_count += 1;
+    return "you lose!";
+  } else if (mypov == "Stone" && computer_pov == "Scissors") {
+    win_count += 1;
+    return "you win!";
+  } else if (mypov == "Stone" && computer_pov == "Paper") {
+    loss_count += 1;
+    return "you lose!";
```

Comment:
This is definitely better for the first time we build and test it, but once we are certain our logic works, we can group all the different winning / losing conditions together, so:

`if ((mypov == "Scissors" && computer_pov == "Paper") 
     || (mypov == "Paper" && computer_pov == "Stone")
     || (mypov == "Stone" && computer_pov == "Scissors")) {
     return "You win!;
 else {  // It doesn't matter what the else condition is, if they didn't win, they lost
 return "You lose!";`